README.md

```diff
@@ -4,84 +4,89 @@ xhgui
+ * [dom](http://php.net/manual/en/book.dom.php) If you are running the tests
+   you'll need the DOM extension (which is a dependency of PHPUnit).
 
 
 Installation
 ============
 
-Installing Xhgui requires 2 main steps. First is installing the `xhgui`
-front-end, and the second is profiling a web application/site.
+1. Clone or download `xhgui` from Github.
 
+2. Point your webserver to the `webroot` directory.
 
-Installing Xhgui
-----------------
+3. Set the permissions on the `cache` directory to allow the
+   webserver to create files. If you're lazy, `0777` will work.
+   
+   The following command changes the permissions for the `cache` directory:
 
-* Clone or download `xhgui` from github.
-* You'll need to install `mongodb`, and `php-mongodb`, at least version 1.3.0
-  of the php extension is required.
-* Point your webserver to the `webroot` directory.
-* Set the permissions on the `cache` cache directory to allow the webserver to
-  create files.  If you're lazy `0777` will work. Run:
+   ```
+   chmod -R 0777 cache
+   ```
 
-  ```
-  chmod -R 0777 cache
-  ```
+4. Start a MongoDB instance. XHGui uses the MongoDB instance to store
+   profiling data.
 
-* If your mongodb setup requires a username + password, or isn't running on the
-  default port + host.  You'll need to update `config/config.php` so that it
-  can connect to mongod.
-* You may wish to add indexes (recommended but optional) to improve the
-  performance, you'll need to do this by using mongo console
+5. If your MongoDB setup uses authentication, or isn't running on the
+   default port and localhost, update XHGui's `config/config.php` so that XHGui
+   can connect to your `mongod` instance.
 
-  On your command prompt (irrespective of Windows or \*nix), open mongo shell
-  using command 'mongo' and follow below  commands to add the index:
+6. (**Optional**, but recommended) Add indexes to MongoDB to improve performance.
 
-  ```
-  $ mongo
-  > use xhprof
-  > db.results.ensureIndex( { 'meta.SERVER.REQUEST_TIME' : -1 } )
-  > db.results.ensureIndex( { 'profile.main().wt' : -1 } )
-  > db.results.ensureIndex( { 'profile.main().mu' : -1 } )
-  > db.results.ensureIndex( { 'profile.main().cpu' : -1 } )
-  > db.results.ensureIndex( { 'meta.url' : 1 } )
-  ```
+   XHGui stores profiling information in a `results` collection in the
+   `xhprof` database in MongoDB. Adding indexes improves performance,
+   letting you navigate pages more quickly.
 
-  After adding indexes, you may notice you can navigate across pages faster.
-* Run the install script. This will download composer and use it to install the
-  dependencies for xhgui.
+   To add an index, open a `mongo` shell from your command prompt.
+   Then, use MongoDB's `db.collection.ensureIndex()` method to add
+   the indexes, as in the following:
 
-    ```
-    cd path/to/xhgui
-    php install.php
-    ```
+   ```
+   $ mongo
+   > use xhprof
+   > db.results.ensureIndex( { 'meta.SERVER.REQUEST_TIME' : -1 } )
```

Comment:
We could mention it. I'm sure someone will get very confused running a new version of MongoDB.